cli/pkg/config/tfvars_generator.go

```diff
@@ -19,7 +19,11 @@ func GenerateTfvars(conf *Config) {
 	vars["ClustersProjectId"] = conf.ClustersProjectID
 	vars["GovernanceProjectId"] = conf.GovernanceProjectID
 	vars["ConfigSync"] = conf.ConfigSync
-	vars["PrivateEndpoint"] = conf.PrivateEndpoint
+	if conf.PrivateEndpoint == true {
```

Comment:
Could you add a comment explaining why this var gets flipped? Thanks!

Reply:
Yep, great idea!